lib/voxpupuli/release/rake_tasks.rb

```diff
@@ -14,6 +14,8 @@
   raise "Refusing to release an RC or build-release (#{v}).\n" +
         "Please set a semver *release* version." unless v =~ /^\d+\.\d+.\d+$/
 
+  # validate that the REFERENCE.md is up2date, if it exists
+  Rake::Task['strings:validate:reference'].invoke if File.exists?('REFERENCE.md')
```

Comment:
`exists?` is deprecated since Ruby 2.2
```suggestion
  Rake::Task['strings:validate:reference'].invoke if File.exist?('REFERENCE.md')
```